src/pages/Checkout/CheckoutPage.vue

```diff
@@ -373,6 +374,21 @@ export default {
 			fragment: experimentVersionFragment,
 		}) || {};
 		this.redirectToLoginExperimentVersion = redirectToLoginExperiment.version;
+
+		// GROW-458 Guest Checkout Experiment
+		const guestCheckoutExperiment = this.apollo.readFragment({
+			id: 'Experiment:guest_checkout',
+			fragment: experimentVersionFragment,
+		}) || {};
+		this.guestCheckoutExperimentVersion = guestCheckoutExperiment.version;
```

Comment:
This should probably be behind the `!cookieStore.get('kvu')` check as well, so that we aren't showing the experiment to existing users